src/main/java/org/cascadebot/cascadebot/utils/PurgeUtils.java

```diff
@@ -0,0 +1,64 @@
+package org.cascadebot.cascadebot.utils;
+
+import net.dv8tion.jda.api.entities.MessageHistory;
+import net.dv8tion.jda.api.entities.Message;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+
+import javax.annotation.Nullable;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class PurgeUtils {
+
+    public enum Criteria {
+        ATTACHMENT,
+        BOT,
+        LINK,
+        TOKEN,
+        USER,
+        ALL,
+        ID,
+    }
+
+    public static void Purge(CommandContext context, Criteria type, int amount, @Nullable String argument) {
```

Comment:
Methods in Java should be written using `lowerCamelCase`

Reply:
Also don't bother putting a @Nullable annotation, we don't use it anywhere else

Reply:
I've heard this one before :/